### PR TITLE
Improve display for 802.1X NAC

### PR DESCRIPTION
**Pull request by PipoCanaja**
- fixed length for port, mac-addr, IP, vlan, username, AuthZ
- tooltip on cells in case the text is too long
- improve the duration display (the 2 bigger units are in the table, and the complete (year, month, days, hours, min, sec) in the tooltip). 

Example:
capture d ecran 2019-01-20 a 00 15 31

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by FTBZ** [comment]
Column AuthZ is too small now:

**Before:**
unknown
**After:**
unknown-1

**Comment by PipoCanaja** [comment]
Interesting. Depends on some Police or browser settings cause it is big enough on mine ... I'll extend it a little bit later today.

**Comment by PipoCanaja** [comment]
@FTBZ You can try again ;) if it does not fit, could you let me know what the minimum value would be for you ?

**Comment by FTBZ** [comment]
Looks good for me.

screenshot 2019-01-21 at 12 37 37

File: html/pages/device/nac.inc.php [hunk 1]
```diff
@@ -69,9 +69,40 @@
         formatters: {
             "time_interval": function (column, row) {
                 var value = row[column.id];
-                return moment.duration(Number(value), 's').humanize();
+                var duration = moment.duration(Number(value), 's');
```
Comment (murrant) [inline html/pages/device/nac.inc.php]: This should probably just be a function in librenms.js

Comment (PipoCanaja) [reply html/pages/device/nac.inc.php]: OK. Will do.

Comment (PipoCanaja) [reply html/pages/device/nac.inc.php]: done

**Pull request merged**